.github/workflows/ci.yml

```diff
@@ -0,0 +1,45 @@
+name: CI
+
+on:
+  push:
+    branches: [ main, master ]
```

Comment:
after we rename master to main, I guess this can be updated?

Reply:
maybe make an issue to track that rename?

Reply:
Yep.  https://github.com/wiki2beamer/wiki2beamer/issues/47